app/views/admin/instances/index.html.slim

```diff
@@ -1,14 +1,12 @@
-h1 = t('.header')
-
-= new_button([:admin, :instance])
-
+= page_header Instance do
+  = new_button([:admin, :instance])
 table.table.table-middle-align
   thead
-  tr
-    th= t('.name')
-    th= t('.host_name')
-    th= t('.users')
-    th= t('.courses')
+    tr
+      th= t('.name')
+      th= t('.host_name')
+      th= t('.users')
+      th= t('.courses')
 
   tbody.table-hover
```

Comment:
Can you check that all tables uses table-hover? And this should apply to the table not the tbody I think..